src/renderer/fonts/font_slant.rs

```diff
@@ -0,0 +1,28 @@
+use skia_safe::font_style::Slant;
+
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub enum FontSlant {
```

Comment:
I've created `FontSlant`, instead using skia Slant directly, just to keep uniform with `FontWeight`and how font things are named at neovim, essentially it doesn't brings any benefits aside the name

Reply:
And i see you did same with FontWeight? or FontWeight is required for something?

I think we should not do this, and just use Weight and Slant from skia directly.

Maybe use `pub use skia_safe::font_style::Weight as FontWeight;`, but in the end we need Skia Weight and Slant, so this is only layer of complexity which could be avoided.

Reply:
I want to avoid using another layer to FontWeight, but, as I've stated at 7208fc4 commit, it doesn't implements Hash, which FontKey requires, and strangely uses `i32` underlying instead `u32`

But I agree with using it to slant as `pub use skia_safe::font_style::Slant as FontSlant;`

Reply:
Hmm, i see. In that case i think it's better to have both structs on neovide side.